src/bazel.rs

```diff
@@ -1203,9 +1129,9 @@ mod tests {
             module.members.iter().any(|(member, _)| member == value)
         }
 
-        // let module = context.get_environment(&LspUrl::File(PathBuf::from("/foo/bar.bzl")));
+        let module = context.get_environment(&LspUrl::File(PathBuf::from("/foo/bar.bzl")));
 
-        // assert!(module_contains(&module, "cc_library"));
+        assert!(module_contains(&module, "cc_library"));
```

Comment:
Won't this mean that bzl files will have `cc_library` as a global suggestion, when it can only be used as `native.cc_library`?

Reply:
This PR does not touch global suggestion - those are handled mostly by `get_environment` method which was not changed, just refactored a litle. The `cc_library` is suggested by `bazel-lsp` also before this PR, and this commented out code was actually passing tests before this PR.

I have changed mostly `parse_file_with_contents` which is responsible for diagnostic. To make it more clear, I have further simplified the code - renamed `go` function to `lint_module` and removed eval.rs completely (as it's not used).